boards/arduino/portenta_h7/arduino_portenta_h7-common.dtsi

```diff
@@ -24,6 +24,7 @@
 		compatible = "usb-ulpi-phy";
 		reset-gpios = < &gpioj 4 GPIO_ACTIVE_LOW >;
 		#phy-cells = <0>;
+		clocks = <&rcc STM32_CLOCK(AHB1, 26)>;
```

Comment:
Part of phy descriptions that are not board specifics should be moved to soc files